ls1intum/Artemis
# [Follow-Up] to #2815: Hidden/private tests

### Checklist

### Description
Follow-up to #2815

- SQL queries are now uppercase (I also reformatted the remaining ones in `ProgrammingExerciseRepository` for consistency and readability)
- Restrain width of dropdowns on ‘Configure Grading’ page (Is there a way to actually wrap the text content?)
![Screenshot_20210329_201834](https://user-images.githubusercontent.com/64250573/112883457-9c026f00-90ce-11eb-85d4-b8b14c1c1551.png)

### Steps for Testing
1. Open the ‘Configure Grading’ page and check that dropdowns are no longer cut off.

### Test Coverage
unchanged
